src/team/team.controller.ts

```diff
@@ -111,22 +114,42 @@ export class TeamController {
 
             if (!teamDoc || !(teamDoc.adminIds).includes(req.person.genesisId)) {
                 log(LOG_LEVEL.INFO, parseLogData(TeamController.TEAM_MESSAGES.UNAUTHORIZED,
-                                                 'TeamController',
-                                                 '401',
-                                                 TeamController.TEAM_MESSAGES.NO_STACK));
+                    'TeamController',
+                    '401',
+                    TeamController.TEAM_MESSAGES.NO_STACK));
 
                 throw new Unauthorized(TeamController.TEAM_MESSAGES.UNAUTHORIZED);
             }
 
+            if (team.contactUserId) {
+                if (typeof (team.contactUserId) !== 'string' && team.contactUserId === "") {
+                    log(LOG_LEVEL.INFO, parseLogData(TeamController.TEAM_MESSAGES.TEAM_CONTACT_USER_ID_INCORRECT,
+                        'TeamController',
+                        '400',
+                        TeamController.TEAM_MESSAGES.NO_STACK));
+
+                    throw new BadRequest(TeamController.TEAM_MESSAGES.TEAM_CONTACT_USER_ID_INCORRECT);
+                }
+
+                if (!teamDoc.userIds.includes(team.contactUserId) && !teamDoc.adminIds.includes(team.contactUserId)) {
+                    log(LOG_LEVEL.INFO, parseLogData(TeamController.TEAM_MESSAGES.USER_NOT_IN_TEAM,
+                        'TeamController',
+                        '400',
+                        TeamController.TEAM_MESSAGES.NO_STACK));
+
+                        throw new BadRequest(TeamController.TEAM_MESSAGES.USER_NOT_IN_TEAM);
```

Comment:
Indentation correctly to the previous line.
```suggestion
                    throw new BadRequest(TeamController.TEAM_MESSAGES.USER_NOT_IN_TEAM);
```